.github/workflows/ci.yml

```diff
@@ -28,9 +28,9 @@ jobs:
     - name: Run visual regression tests
       run: CI=true npm run test.browser
 
-    - name: Update regression test snapshots
-      if: ${{ failure() }}
-      run: npm run test.browser -- -u
+    #- name: Update regression test snapshots
+      #if: ${{ failure() }}
+      #run: npm run test.browser -- -u
```

Comment:
Don't we need to update these visual snapshots in order to get tests to pass?

Reply:
Yes!